aldourtusuastegui/MovieTime
# Android | Feature/movies |Display PopularMovies, RatedMovies, RecommendedMovies

### **What is solving?** 
Display PopularMovies, RatedMovies, RecommendedMovies

### **Solution description**
-created feature_movies with clean architecture and MVVM.
-data is obtained from api.
-created adapter for recyclerView.
-displayed movies.

### **Steps to verify changes**
-Run the application or see attached image

<img src="https://github.com/aldourtusuastegui/MovieTime/assets/13209412/c6dcbd97-3c16-40e7-a5bf-90b331197d63"  width="300" height="600" />
